plugins/lookup/random_pet.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Abhijeet Kasurde <akasurde@redhat.com>
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    name: random_pet
+    author:
+      - Abhijeet Kasurde (@Akasurde)
+    short_description: Generates random pet names
+    version_added: '3.0.2'
+    requirements:
+      - Python-petname U(https://github.com/dustinkirkland/python-petname).
```

Comment:
```suggestion
      - petname U(https://github.com/dustinkirkland/python-petname).
```
After all it's `pip install petname`.